src/Comments.php

```diff
@@ -51,9 +61,7 @@ public function delete(string ...$comments): self
         foreach ($comments as $comment) {
             $key = array_search($comment, $this->comments);
 
-            if (is_int($key)) {
```

Comment:
My [previous question](https://github.com/php-gettext/Gettext/pull/307/files#r2071832898) was more related to this change.

As far as I understand, should `Comments::$comments` be an array of integer keys only? If that is correct, this condition would be unnecessary :)

Reply:
`$key` can be false, if the comment wasn't found.